README.md

```diff
@@ -68,12 +68,15 @@ libsecp256k1 is built using autotools:
     $ make check  # run the test suite
     $ sudo make install  # optional
 
+To compile optional modules (such as Schnorr), you need to run `./configure` with additional flags (such as `--enable-module-schnorr`). Run `./configure --help` to see the full list of available flags.
```

Comment:
```suggestion
To compile optional modules (such as Schnorr signatures), you need to run `./configure` with additional flags (such as `--enable-module-schnorrsig`). Run `./configure --help` to see the full list of available flags.
```

Also below.

"Schnorr" is a person, so I think "enable Schnorr" is very strange. I keep insisting on this point. (But I don't always win  -- https://github.com/rust-bitcoin/rust-secp256k1/pull/327)